dev/com.ibm.ws.microprofile.health.2.0/src/com/ibm/ws/microprofile/health20/internal/HealthCheck20ServiceImpl.java

```diff
@@ -107,7 +107,7 @@ public void performHealthCheck(HttpServletRequest request, HttpServletResponse h
             if (!appTracker.isStarted(appName)) {
                 if (tc.isDebugEnabled())
                     Tr.debug(tc, "In performHealthCheck(): Application : " + appName + " has not started yet.");
-                if (healthCheckProcedure == HealthCheckConstants.HEALTH_CHECK_READY || healthCheckProcedure == HealthCheckConstants.HEALTH_CHECK_ALL) {
+                if (healthCheckProcedure != HealthCheckConstants.HEALTH_CHECK_LIVE) {
```

Comment:
Comparing Strings with != and == ?

Reply:
Forgot to push that change in, good catch!